src/TouristInfoCards.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div className="card-container">
+      {props.cities.map((city, key) => (
+        <div className="card" key={key}>
```

Comment:
It is often best to not use the array index as the key here. 
Take a look at the keys sections in the docs, there is a better explanation than I could ever give 😊  https://legacy.reactjs.org/docs/lists-and-keys.html